lib/humidifier/version.rb

```diff
@@ -1,4 +1,4 @@
 module Humidifier
   # Gem version
-  VERSION = '0.3.0'.freeze
+  VERSION = '0.3.1'.freeze
```

Comment:
We might want to think more about the versioning strategy here--some of the changes below, like the changes from string to JSON object, are breaking. How will we handle this moving forward?